h3/sf/sql/VERSION.sql

```diff
@@ -0,0 +1,12 @@
+-----------------------------------------------------------------------
+--
+-- Copyright (C) 2021 CARTO
+--
+-----------------------------------------------------------------------
+
+CREATE OR REPLACE SECURE FUNCTION @@SF_DATABASEID@@.@@SF_SCHEMA_H3@@.VERSION()
+    RETURNS STRING
+    LANGUAGE JAVASCRIPT
+AS $$
+    return '3.7.0.0';
```

Comment:
We should use a method here: h3Version()

Reply:
I have done it in one of my latest commits.
We were using directly the library "h3-js.umd.js" and now put together with our custom code, as we do in other modules.